packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -9,9 +9,11 @@
   $govuk-small-checkboxes-size: 24px;
   $govuk-checkboxes-label-padding-left-right: govuk-spacing(3);
 
-  .govuk-checkboxes__item {
+  .govuk-checkboxes {
     @include govuk-font($size: 19);
+  }
```

Comment:
Hear me out @owenatgov, I'm being controversial

We already set `font-size` on the Checkbox label and hint, so can we ditch this? 😱

I guess it's only needed because we use a `<div>` for the divider text?

```html
<div class="govuk-checkboxes__divider">or</div>
```

```suggestion
```

Reply:
Good spot! This also applies to radios. I tried removing the `govuk-text-colour` as well and `govuk-font` doesn't apply this 😬 I'm gonna do a second pass to double check none of these have lost our off-black.

Reply:
Only impact was on the summary card, which I'm pretty sure has never had the proper colour 😱 Fixed as part of this.